LottieSample/src/main/kotlin/com/airbnb/lottie/samples/LottiefilesViewModel.kt

```diff
@@ -53,6 +53,7 @@ class LottiefilesViewModel(application: Application) : AndroidViewModel(applicat
         disposables.add(observable
                 .subscribeOn(Schedulers.io())
                 .observeOn(AndroidSchedulers.mainThread())
+                .retry(3)
```

Comment:
some country visit [https://www.lottiefiles.com](https://www.lottiefiles.com) spent a lot of time, and often occur SocketTimeoutException, so I want retry 3 times when occur exception.